i18n/es.jsonp.js

```diff
@@ -24,7 +25,7 @@ document.localeJson = {
       "configure-changed-content-note": "(Vuelva a seleccionar carpeta si hay contenido actualizado en su dispositivo)",
       "configure-btn-rescan": "Volver a escanear",
       "configure-about-rescan-tip": "Volver a escanear las tarjetas SD y la memoria interna",
-      "configure-expert-enable-source-verification-tip": "Advertencia: Algunos archivos ZIM de fuentes no fiables podrían ejecutar código malicioso en tu navegador. Esto puede evitarse utilizando el modo JQuery, que no puede ejecutar contenido activo desde el ZIM. Los ZIM muy dinámicos probablemente fallarán en el modo JQuery, pero los ZIM con contenido mayoritariamente estático deberían funcionar. Si confía en la fuente de todos sus archivos ZIM, puede desactivar esta opción y se utlizará el modo ServiceWorker por defecto (si está disponible).",
+      "configure-expert-enable-source-verification-tip": "Advertencia: Algunos archivos ZIM de fuentes no fiables podrían ejecutar código malicioso en tu navegador. Esto puede evitarse utilizando el modo Seguro, que no puede ejecutar contenido activo desde el ZIM. Los ZIM muy dinámicos probablemente fallarán en el modo Seguro, pero los ZIM con contenido mayoritariamente estático deberían funcionar. Si confía en la fuente de todos sus archivos ZIM, puede desactivar esta opción y se utlizará el modo ServiceWorker por defecto (si está disponible).",
```

Comment:
I've just noticed that "en tu navegador" should be "en su navegador". (The former is a familiar form of your, but throughout the app we use a more formal form.)